styles/style.css

```
@@ -0,0 +1,12 @@
+header{
```

It's not a good practice grab itens by the element name. You can add a class to the element, can be "header", and here use the class selector to grab it.
```suggestion
.header {
```